res/sections/16-Les21032017.tex

```diff
@@ -220,13 +221,27 @@ \subsection{Biometria}
 Chi sei e cosa fai, è suscettibile ad errori.
 
 \begin{itemize}
-\item \textbf{False Rejection Rate:} anche detto \textbf{falso negativo}, un 
-reject che non dovrebbe esserci stato;
-\item \textbf{False Acceptance Rate:} anche detto \textbf{falso positivo}, un 
-accept quando non avrebbe dovuto esserci;
-\item \textbf{Failure to Enroll Rate:} numero di utenti che non sono riusciti a
-registrarsi correttamente.
+\item \textbf{False Rejection Rate} (FRR): anche detto 
```

Comment:
Si ma l'EE non esiste ^^ senza FAR e FRR. Teniamo cosí che è in linea con il libro:

> False rejection rate (FRR), type 1 error  ​ ​Used in biometrics, false rejection means to
reject access to an authorized user by mistake. This is the most common type of biometric
authentication error.
e 
> False acceptance rate (FAR), type 2 error  ​ ​A n error condition in biometrics that grants
an unwanted user with permission to access the system by mistake. This is a less common
error since falsely granting access to an unauthorized person is supposed to be rare.